README.rst

```diff
@@ -5,6 +5,75 @@ plone.staticresources
 Static JS and CSS resources for Plone.
 
 
+This add on contains all JavaScript and CSS resources from Plone.
+Prior to Plone 5.2, they were located in Products.CMFPlone.
+
+.. note::
+  A note on yarn: We require the use of yarn, because yarn supports installing the packages in a custom directory, which npm doesn't.
+  Our package directory is located at: ``src/plone/staticresources/static/components``.
+
+
+How to develop on patterns
+--------------------------
+
+All JavaScript code in this package is downloaded via yarn into ``src/plone/staticresources/static/components``.
+Nothing in this directory should be touched.
+If you need to fix something, fix it in the original repository and eventually upgrade the version (described below).
+For Mockup, the original repository is: https://github.com/plone/mockup/
+For Patternslib, visit: http://github.com/patternslib/Patterns
+
+
+How to upgrade the resources in this package
+--------------------------------------------
+
+- Increase the npm package versions in package.json.
+- Run ``yarn upgrade``.
+- Run ``./bin/plone-compile-resources -b plone``, ``./bin/plone-compile-resources -b plone-logged-in`` and whatever bundle you're going to build in your buildout's root directory.
+  For more info on ``plone-compile-resources`` see: XXX
```

Comment:
I guess this `XXX` is bound to be changed, right? :wink:

Reply:
I believe so :) And used the same convention in a new commit. It's a WIP and your all the feedback has been very useful! Please continue so we fill the gaps.